ignite/contrib/handlers/mlflow_logger.py

```diff
@@ -173,10 +290,10 @@ class OptimizerParamsHandler(BaseOptimizerParamsHandler):
         tag (str, optional): common title for all produced plots. For example, 'generator'
     """
 
-    def __init__(self, optimizer, param_name="lr", tag=None):
+    def __init__(self, optimizer: Optimizer, param_name: str = "lr", tag: Optional[str] = None):
         super(OptimizerParamsHandler, self).__init__(optimizer, param_name, tag)
 
-    def __call__(self, engine, logger, event_name):
+    def __call__(self, engine: Engine, logger: MLflowLogger, event_name):
```

Comment:
`event_name: str`

Reply:
@gruebel `event_name` is `Union[str, EventEnum]`, https://github.com/pytorch/ignite/blob/8d8ca990f38b113652cf3565920ec38ef5508ba5/ignite/engine/engine.py#L144 
Maybe, it would make sense to introduce type alias for that as `EventType` or something.

btw, thanks for your help with the review !

Reply:
Done, changed to `Union[str, EventEnum]`